tests/login.spec.js

```diff
@@ -0,0 +1,28 @@
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+});
+
+test('login', async ({ page }) => {
+    await page.goto('https://www.saucedemo.com/');
+
+    //login
+    await page.getByPlaceholder('Username').click();
+    await page.getByPlaceholder('Username').fill('standard_user');
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill('secret_sauce');
```

Comment:
Let's not hardcode the login creads here. Lets move it to the separate file and use it here